### PR TITLE
Should render suggestions - AutocompleteArrayInput

### PR DESCRIPTION
**Pull request by davidpicarra**
Following #2502, this PR allows for the creator of an `<AutocompleteArrayInput />` to pass in an optional `shouldRenderSuggestions` function.

Was trying to set this function for the `<AutocompleteArrayInput />` and noticed that #2502 only adds this functionality to `<AutocompleteInput />`.

### CODE REVIEW

**Comment by fzaninotto** [comment]
Could you please PR against `next` instead of `master`, as it's a new feature?

File: packages/ra-ui-materialui/src/input/AutocompleteInput.js [hunk 1]
```diff
@@ -449,7 +449,7 @@ export class AutocompleteInput extends React.Component {
         ) {
             return shouldRenderSuggestions(val);
         }
-
+
```
Comment (djhi) [inline packages/ra-ui-materialui/src/input/AutocompleteInput.js]: Can you revert this ?

Comment (davidpicarra) [reply packages/ra-ui-materialui/src/input/AutocompleteInput.js]: My bad, should be fixed now

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**